services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/application/ApplicationServiceITest.kt

```diff
@@ -597,6 +598,58 @@ class ApplicationServiceITest : DatabaseTest() {
             }
         }
 
+        @Test
+        fun `when application is in Allu should initialize access for new contacts`() {
+            val cableReport =
+                createCableReportApplicationData(
+                    areas = listOf(aleksanterinpatsas),
+                    customerWithContacts = hakijaCustomerContact,
+                    contractorWithContacts = suorittajaCustomerContact,
+                )
+            val application =
+                hankeService.generateHankeWithApplication(
+                    CableReportWithoutHanke(CABLE_REPORT, cableReport),
+                    USERNAME,
+                )
+            mockSendAllu(applicationRepository.findById(application.id!!).orElseThrow())
+            val capturedEmails = mutableListOf<ApplicationNotificationData>()
```

Comment:
Maybe capturedNotifications? To make it clearer these don't include the invitations.

Reply:
Fixed.